httpx_ws/_api.py

```diff
@@ -1078,40 +1078,114 @@ def _get_headers(
     return headers
 
 
-@contextlib.contextmanager
-def _connect_ws(
-    url: str,
-    client: httpx.Client,
-    *,
-    max_message_size_bytes: int = DEFAULT_MAX_MESSAGE_SIZE_BYTES,
-    queue_size: int = DEFAULT_QUEUE_SIZE,
-    keepalive_ping_interval_seconds: typing.Optional[
-        float
-    ] = DEFAULT_KEEPALIVE_PING_INTERVAL_SECONDS,
-    keepalive_ping_timeout_seconds: typing.Optional[
-        float
-    ] = DEFAULT_KEEPALIVE_PING_TIMEOUT_SECONDS,
-    subprotocols: typing.Optional[list[str]] = None,
-    session_class: type[SyncSession] = WebSocketSession,  # type: ignore[assignment]
-    **kwargs: typing.Any,
-) -> typing.Generator[SyncSession, None, None]:
```

Comment:
The internal implementation function `_[a]connect_ws` has been replaced with the implementation of the `[Async]WebSocketClient.connect` method. This change was made to avoid redundant calls caused by keeping `_[a]connect_ws`.